node/service/src/chain_spec/moonbase.rs

```diff
@@ -41,8 +41,7 @@ use sp_core::ecdsa;
 use sp_runtime::{traits::One, Perbill, Percent};
 
 /// Specialized `ChainSpec`. This is a specialization of the general Substrate ChainSpec type.
-pub type ChainSpec =
-	sc_service::GenericChainSpec<RuntimeGenesisConfig, Extensions, crate::HostFunctions>;
+pub type ChainSpec = sc_service::GenericChainSpec<Extensions>;
```

Comment:
Host functions should be kept in the ChainSpec because of `moonbeam_primitives_ext::moonbeam_ext::HostFunctions`
```suggestion
pub type ChainSpec = sc_service::GenericChainSpec<Extensions, crate::HostFunctions>;
```